### ISSUE TITLE
A way of creating directory if it doesn't exist?

### ISSUE THREAD

**Issue by samedii** [AvianFlu/ncp#54]
Like the --parent option in cp

**Comment by GochoMugo** [comment]
:+1: really looking forward for this feature. This will allow **not** requiring to wrap `ncp()` with `fs.mkdir()` _(or something alike)_.

**Comment by pgangwani** [comment]
Hello Guys, I need to get the fix like it asked here, Please help

**Comment by Jonahss** [comment]
There's so many unmerged pull requests on this repo. maybe it's time for somebody to fork this repo, merge them all, and publish.

**Comment by pgangwani** [comment]
@AvianFlu @Jonahss : Can you guys please help

**Comment by Jonahss** [comment]
Well. I can help advise, but I don't have a whole workday to devote to this.
@pgangwani are you a javascript developer?

**Comment by pgangwani** [comment]
Yes , i am . I couldn't find any PR in the forked branch .
May be need some one to review most of important and merge.

**Comment by Jonahss** [comment]
Ah yes. So your forked repo won't have these fine people's pull requests, since they didn't request YOU for the pull request :)

take a look at this graph: https://github.com/AvianFlu/ncp/network
That's all the people who have forked this repo and added their own changes. I suggest u look through them all (or the list of PRs on this repo), add their repos as remotes, and merge all their changes into one updated version of this repo :)

If you do that successfully, we may be able to petition npm for the rights to this module, since the author seems to have dropped off the map.

**Comment by Jonahss** [comment]
@pgangwani Did you find a solution? Need any help?

**Comment by pgangwani** [comment]
Unfortunately, I didn't get time, I  am adding you as collabrator , Please
help there

**Comment by pgangwani** [comment]
@Jonahss : Are you seeing my merges

**Comment by Jonahss** [comment]
Uh no. can you send me a link to what you're talking about?

**Comment by pgangwani** [comment]
https://github.com/pgangwani/ncp

**Comment by Jonahss** [comment]
Cool. I would suggest running tests and making sure the code works after merging all those.

**Comment by pgangwani** [comment]
OK , I am merging so many PRs for first time, can you run whatever I have
done till now.

**Comment by pgangwani** [comment]
Do I need to configure test somewhere for new PRs

**Comment by Jonahss** [comment]
I don't quite understand your question. Basically, you should read all the code for this module, evaluate various pull requests, merge the ones you think belong in the module. make sure it works, make sure there is adequate test coverage, then publish a new module that you are happy with.

If the module is an improvement on the original, you can suggest a pull request to this repo, or publish on npm under a different name.

If you think the community will find your improved version more useful, we can petition npm to take the module name from this outdated repo.

**Comment by pgangwani** [comment]
Yeah, thanks man, I could manage to merge 16 , 26 more to review.

**Comment by Jonahss** [comment]
Cool!
